stdlib/public/SDK/CoreGraphics/CMakeLists.txt

```diff
@@ -8,6 +8,6 @@ add_swift_library(swiftCoreGraphics ${SWIFT_SDK_OVERLAY_LIBRARY_BUILD_TYPES} IS_
   SWIFT_COMPILE_FLAGS "${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS}"
   LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
   SWIFT_MODULE_DEPENDS ObjectiveC Dispatch Darwin
-  SWIFT_MODULE_DEPENDS_OSX IOKit XPC
+  SWIFT_MODULE_DEPENDS_OSX Dispatch IOKit ObjectiveC os
```

Comment:
I am going off of the ``TARGET_SDKS OSX IOS IOS_SIMULATOR TVOS TVOS_SIMULATOR
`` lines in the ``CMakeLists.txt`` files for each overlay. ``CoreGraphics`` doesn't have a ``TARGET_SDKS`` line so I'm reluctant to add more platforms. I'm just refactoring--someone who knows which platforms each supports should do it. (Or I can, if I learn how to tell which overlays support which platforms).